frontend/src/components/SeedlotRegistrationProgress/InterimStorage/index.tsx

```diff
@@ -0,0 +1,371 @@
+import React, { useState, useRef, useEffect } from 'react';
+// import { useNavigate } from 'react-router-dom';
+import axios from 'axios';
+
+import {
+  Button,
+  Checkbox,
+  Column,
+  ComboBox,
+  DatePicker,
+  DatePickerInput,
+  NumberInput,
+  RadioButton,
+  RadioButtonGroup,
+  Row,
+  TextInput
+} from '@carbon/react';
+
+import { useParams } from 'react-router-dom';
+import { ArrowRight } from '@carbon/icons-react';
+import { FilterObj, filterInput } from '../../../utils/filterUtils';
+import Subtitle from '../../Subtitle';
+import InterimStorageRegistration from '../../../types/InterimStorageRegistration';
+import getUrl from '../../../utils/ApiUtils';
+import ApiAddresses from '../../../utils/ApiAddresses';
+import { useAuth } from '../../../contexts/AuthContext';
+import ApplicantInfo from '../../../types/ApplicantInfo';
+import './styles.scss';
+
+interface InterimStorageStepProps {
+  setStep: Function
+}
+
+const InterimStorage = ({ setStep }: InterimStorageStepProps) => {
+  const { token } = useAuth();
+  const { seedlot } = useParams();
+
+  const mockAgencyOptions: Array<string> = [
+    '0032 - Strong Seeds Orchard - SSO',
+    '0035 - Weak Seeds Orchard - WSO',
+    '0038 - Okay Seeds Orchard - OSO'
+  ];
+
+  const mockStartDataMin: string = '03/20/2023';
+  const mockStartDataMax: string = '03/23/2023';
+  const mockEndDataMin: string = '03/20/2023';
+  const mockEndDataMax: string = '03/23/2023';
```

Comment:
![image](https://user-images.githubusercontent.com/25162561/227052696-d80a0eaa-92b0-4272-9219-3b1f9feee18c.png)
We should not use mock data here, we need to implement actual logic to enforce the rule stated in the photo